Makefile

```diff
@@ -125,49 +137,49 @@ release-test: release-tester ## Test the currently selected release package
 docker-image-build-12 docker-image-build-13 docker-image-build-14: Dockerfile $(SQL_FILES) $(SRCS) Cargo.toml Cargo.lock $(RUST_SRCS)
 	docker buildx build $(if $(PUSH),--push,--load) \
 		--build-arg TIMESCALEDB_VERSION=$(TIMESCALEDB_VER) \
-		--build-arg PG_VERSION_TAG=$(PG_VER) \
-		-t $(IMAGE_NAME):$(EXT_VERSION)-$(TIMESCALEDB_VER)-$(PG_VER) \
-		-t $(IMAGE_NAME):$(EXT_VERSION)-ts$(TIMESCALEDB_MAJOR)-$(PG_VER) \
-		-t $(IMAGE_NAME):latest-ts$(TIMESCALEDB_MAJOR)-$(PG_VER) \
+		--build-arg PG_VERSION_TAG=$(PG_BUILD_VER) \
+		-t $(IMAGE_NAME):$(EXT_VERSION)-$(TIMESCALEDB_VER)-$(PG_BUILD_VER) \
+		-t $(IMAGE_NAME):$(EXT_VERSION)-ts$(TIMESCALEDB_MAJOR)-$(PG_BUILD_VER) \
+		-t $(IMAGE_NAME):latest-ts$(TIMESCALEDB_MAJOR)-$(PG_BUILD_VER) \
 		.
 
 .PHONY: docker-image-12
-docker-image-12: PG_VER=pg12
+docker-image-12: PG_BUILD_VER=pg12
```

Comment:
I find it somewhat confusing that `make release` is a single target and relies on passing `OS_NAME`, `OS_VERSION`, and `PG_RELEASE_VERSION` variables, yet `make docker-image` is a group of targets and they manipulate `PG_BUILD_VER` internally.